java/src/main/java/search/BinarySearch.java

```diff
@@ -4,14 +4,14 @@
 import java.util.Arrays;
 
 public class BinarySearch {
+    // Array is assumed to be sorted for the purposes of binary search
```

Comment:
Good fix, however I believe this should have been in a separate PR :sweat_smile: